cluster/sbin/snappy-nodes.sh

```diff
@@ -141,6 +143,8 @@ function startComponent() {
     ssh $SPARK_SSH_OPTS "$host" $"${@// /\\ } ${args}" < /dev/null \
       2>&1 | sed "s/^/$host: /"
   fi
+
+  echo "${host} ${dirfolder}" >> $MEMBERS_FILE
```

Comment:
I think it will be better for this to into work dir instead of current dir (i.e. "$SPARK_HOME/work")